### PR TITLE
add OrganizationTile

### LINKED ISSUE #100: Organization tile
Organization Card_active

### PR DESCRIPTION
**Pull request by DasProffi**
closes #100
Unbenannt

### CODE REVIEW

**Review by JannikWibker — APPROVED** [review]
as far as I can tell from the screenshot provided the font sizes and font weights might need a little bit of work, this could also just a design issue, but I think we can merge this and do a bit of clean up later on

File: tasks/components/OrganizationTile.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    <Card onTileClick={onTileClick} isSelected={isSelected} className={tw('group cursor-pointer')}>
+      <div className={tw('flex flex-row justify-between w-full')}>
+        <span className={tw('font-bold font-space')}>{`${organization.longName} (${organization.shortName})`}</span>
+        <button onClick={onEditClick}
+                className={tw('hidden group-hover:block')}>
+            {translation.edit}
+        </button>
+      </div>
+      <div className={tw('text-left my-1 font-semibold text-gray-600 text-sm')}>
+        {organization.wards.map(value => value.name).join(', ')}
```
Comment (MaxSchaefer) [inline tasks/components/OrganizationTile.tsx:63]: We should limit the amount of wards, as we already do with the Members in the AvatarGroup.

**Review by MaxSchaefer — CHANGES_REQUESTED** [review]
I've got the following Bug (no AvatarGroup).

Bildschirm­foto 2023-03-25 um 3 23 12 PM

```tsx
import type { NextPage } from 'next'
import { OrganizationTile } from '../components/OrganizationTile'

const Test: NextPage = () => {
  return (
    <div className={'flex'}>
      <OrganizationTile organization={{
        email: 'info@ukm.de',
        longName: 'Uniklinikum Münster',
        shortName: 'UKM',
        members: [
          {
            name: 'One Two',
            avatarURL: 'https://source.boringavatars.com/'
          },
          {
            name: 'Three Four',
            avatarURL: 'https://source.boringavatars.com/'
          },
          {
            name: 'Five Six',
            avatarURL: 'https://source.boringavatars.com/'
          },
          {
            name: 'Seven Eight',
            avatarURL: 'https://source.boringavatars.com/'
          },
          {
            name: 'Nine Ten',
            avatarURL: 'https://source.boringavatars.com/'
          },
        ],
        wards: [
          { name: 'ICU#1' }, { name: 'ICU#2' }
        ]
      }}/>
      <OrganizationTile organization={{
        email: 'info@ukm.de',
        longName: 'Uniklinikum Münster',
        shortName: 'UKM',
        members: [
          {
            name: 'One Two',
            avatarURL: 'https://source.boringavatars.com/'
          },
          {
            name: 'Three Four',
            avatarURL: 'https://source.boringavatars.com/'
          },
          {
            name: 'Five Six',
            avatarURL: 'https://source.boringavatars.com/'
          },
          {
            name: 'Seven Eight',
            avatarURL: 'https://source.boringavatars.com/'
          },
          {
            name: 'Nine Ten',
            avatarURL: 'https://source.boringavatars.com/'
          },
        ],
        wards: [
          { name: 'ICU#1' }, { name: 'ICU#2' }
        ]
      }}/>
      <OrganizationTile organization={{
        email: 'info@ukm.de',
        longName: 'Uniklinikum Münster',
        shortName: 'UKM',
        members: [
          {
            name: 'One Two',
            avatarURL: 'https://source.boringavatars.com/'
          },
          {
            name: 'Three Four',
            avatarURL: 'https://source.boringavatars.com/'
          },
          {
            name: 'Five Six',
            avatarURL: 'https://source.boringavatars.com/'
          },
          {
            name: 'Seven Eight',
            avatarURL: 'https://source.boringavatars.com/'
          },
          {
            name: 'Nine Ten',
            avatarURL: 'https://source.boringavatars.com/'
          },
        ],
        wards: [
          { name: 'ICU#1' }, { name: 'ICU#2' }
        ]
      }}/>
      <OrganizationTile organization={{
        email: 'info@ukm.de',
        longName: 'Uniklinikum Münster',
        shortName: 'UKM',
        members: [
          {
            name: 'One Two',
            avatarURL: 'https://source.boringavatars.com/'
          },
          {
            name: 'Three Four',
            avatarURL: 'https://source.boringavatars.com/'
          },
          {
            name: 'Five Six',
            avatarURL: 'https://source.boringavatars.com/'
          },
          {
            name: 'Seven Eight',
            avatarURL: 'https://source.boringavatars.com/'
          },
          {
            name: 'Nine Ten',
            avatarURL: 'https://source.boringavatars.com/'
          },
        ],
        wards: [
          { name: 'ICU#1' }, { name: 'ICU#2' }
        ]
      }}/>
    </div>
  )
}

export default Test
```

File: tasks/components/OrganizationTile.tsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+type OrganizationDTO = {
+  longName: string,
+  shortName: string,
+  wards: WardDTO[],
+  email: string,
+  members: UserDTO[]
+}
+
+export type OrganizationTileProps = CardProps & {
+  displayedAvatars?: number,
```
Comment (MaxSchaefer) [inline tasks/components/OrganizationTile.tsx:47]: This should be `maxShownWards` also to stay consistent with the `AvatarGroup`.

**Review by MaxSchaefer — CHANGES_REQUESTED** [review]

**Comment by MaxSchaefer** [comment]
> I've got the following Bug (no AvatarGroup).
> 
> Bildschirm­foto 2023-03-25 um 3 23 12 PM
> 
> ```tsx
> import type { NextPage } from 'next'
> import { OrganizationTile } from '../components/OrganizationTile'
> 
> const Test: NextPage = () => {
>   return (
>     <div className={'flex'}>
>       <OrganizationTile organization={{
>         email: 'info@ukm.de',
>         longName: 'Uniklinikum Münster',
>         shortName: 'UKM',
>         members: [
>           {
>             name: 'One Two',
>             avatarURL: 'https://source.boringavatars.com/'
>           },
>           {
>             name: 'Three Four',
>             avatarURL: 'https://source.boringavatars.com/'
>           },
>           {
>             name: 'Five Six',
>             avatarURL: 'https://source.boringavatars.com/'
>           },
>           {
>             name: 'Seven Eight',
>             avatarURL: 'https://source.boringavatars.com/'
>           },
>           {
>             name: 'Nine Ten',
>             avatarURL: 'https://source.boringavatars.com/'
>           },
>         ],
>         wards: [
>           { name: 'ICU#1' }, { name: 'ICU#2' }
>         ]
>       }}/>
>       <OrganizationTile organization={{
>         email: 'info@ukm.de',
>         longName: 'Uniklinikum Münster',
>         shortName: 'UKM',
>         members: [
>           {
>             name: 'One Two',
>             avatarURL: 'https://source.boringavatars.com/'
>           },
>           {
>             name: 'Three Four',
>             avatarURL: 'https://source.boringavatars.com/'
>           },
>           {
>             name: 'Five Six',
>             avatarURL: 'https://source.boringavatars.com/'
>           },
>           {
>             name: 'Seven Eight',
>             avatarURL: 'https://source.boringavatars.com/'
>           },
>           {
>             name: 'Nine Ten',
>             avatarURL: 'https://source.boringavatars.com/'
>           },
>         ],
>         wards: [
>           { name: 'ICU#1' }, { name: 'ICU#2' }
>         ]
>       }}/>
>       <OrganizationTile organization={{
>         email: 'info@ukm.de',
>         longName: 'Uniklinikum Münster',
>         shortName: 'UKM',
>         members: [
>           {
>             name: 'One Two',
>             avatarURL: 'https://source.boringavatars.com/'
>           },
>           {
>             name: 'Three Four',
>             avatarURL: 'https://source.boringavatars.com/'
>           },
>           {
>             name: 'Five Six',
>             avatarURL: 'https://source.boringavatars.com/'
>           },
>           {
>             name: 'Seven Eight',
>             avatarURL: 'https://source.boringavatars.com/'
>           },
>           {
>             name: 'Nine Ten',
>             avatarURL: 'https://source.boringavatars.com/'
>           },
>         ],
>         wards: [
>           { name: 'ICU#1' }, { name: 'ICU#2' }
>         ]
>       }}/>
>       <OrganizationTile organization={{
>         email: 'info@ukm.de',
>         longName: 'Uniklinikum Münster',
>         shortName: 'UKM',
>         members: [
>           {
>             name: 'One Two',
>             avatarURL: 'https://source.boringavatars.com/'
>           },
>           {
>             name: 'Three Four',
>             avatarURL: 'https://source.boringavatars.com/'
>           },
>           {
>             name: 'Five Six',
>             avatarURL: 'https://source.boringavatars.com/'
>           },
>           {
>             name: 'Seven Eight',
>             avatarURL: 'https://source.boringavatars.com/'
>           },
>           {
>             name: 'Nine Ten',
>             avatarURL: 'https://source.boringavatars.com/'
>           },
>         ],
>         wards: [
>           { name: 'ICU#1' }, { name: 'ICU#2' }
>         ]
>       }}/>
>     </div>
>   )
> }
> 
> export default Test
> ```

Issue fixed! Thanks 👍

**Review by MaxSchaefer — APPROVED** [review]

**Pull request merged**